examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -248,22 +244,15 @@
     "                    timestamps=np.concatenate(timestamps),\n",
     "                    agent_ids=np.concatenate(agent_ids))\n",
     "\n",
-    "# get a pretty visualisation of the errors\n",
-    "table = PrettyTable(field_names=[\"future step\", \"MSE\"])\n",
-    "table.float_format = \".2\"\n",
-    "steps = range(1, cfg[\"model_params\"][\"future_num_frames\"] + 1)\n",
-    "errors = compute_mse_error_csv(gt_path, pred_path)\n",
```

Comment:
mmm this shouldn't be removed! (it was recently introduced in master)

Reply:
Weird, I did a fetch and merge before pushing, I'll check at once and can open a new PR (since I can't edit this one, role request has still not come through)

Reply:
Fixed now. Check the newest commit (right under this review)